src/observations/test_trend_observation_rms.py

```
@@ -0,0 +1,150 @@
+    trend_observation_rms.trend_history = deque(
+        [
+            np.array([1.0, 2.0, 3.0]),
+            np.array([2.0, 3.0, 4.0]),
+            np.array([3.0, 4.0, 5.0]),
+        ]
+    )
+
+    # Calculate the expected RMS
+    all_values = [1.0, 2.0, 3.0, 2.0, 3.0, 4.0, 3.0, 4.0, 5.0]
+    expected_rms = np.sqrt(np.mean(np.square(all_values)))
+    expected_multiplier = 1 / expected_rms
```

Ruff

Local variable `expected_multiplier` is assigned to but never used

Remove assignment to unused variable `expected_multiplier`

(F841)